test/cors_plug_test.exs

```diff
@@ -14,7 +14,8 @@ defmodule CORSPlugTest do
 
   test "lets me overwrite options" do
     opts = CORSPlug.init(origin: "example.com")
-    conn = conn(:get, "/", nil, headers: [{"origin", "example.com"}])
+    conn = conn(:get, "/")
+    |> put_req_header("origin", "example.com")
```

Comment:
@mschae thanks! agreed, mind if i also start the pipeline with bare value (e.g., via [elixir style guide](https://github.com/rrrene/elixir-style-guide) and [dogma](https://github.com/lpil/dogma/blob/master/docs/rules.md#pipelinestart)):
```
conn = 
  :get
  |> conn("/")
  |> put_req_header("origin", "example.com")
```